test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -40,6 +40,28 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $ps.Dispose()
     }
 
+    It "Should accept minimum depth as 0." {
+        $ComplexObject = [PSCustomObject] @{
+            FirstLevel1  = @{
```

Comment:
Can you include a value type in the test?

Reply:
@TravisEz13  Done